index.js

```diff
@@ -0,0 +1,120 @@
+const inquirer = require("inquirer");
+const figlet = require('figlet');
+const mysql = require("mysql2");
+const cTable = require("console.table");
```

Comment:
nice job organising all your external imports at the top - great for readability :)